addons/mail/models/mail_message.py

```diff
@@ -563,7 +563,7 @@ def _make_access_error(self, operation: str) -> AccessError:
             "Records: %(records)s, User: %(user)s",
             type=self._description,
             operation=operation,
-            records=format_list(self.env, list(map(str, self.ids[:6]))),
+            records=list(map(str, self.ids[:6])),
```

Comment:
Both `list` and `map(str` are unnecessary with the new implementation (it's built in)

Reply:
Good catch 👌 Updated it.